src/App.js

```diff
@@ -22,6 +23,21 @@ function App() {
     setToken(newToken);
   };
 
+  const shareToken = (e, token) => {
+    e.preventDefault();
+    db.collection(token)
+      .get()
+      .then((resp) => {
+        if (resp.size) {
+          localStorage.setItem('Token', token);
+          setToken(token);
+          console.log('Im valid!');
```

Comment:
Console logs like these are great for testing but we should try to remove them when merging code to main :D